definitions/variable/macro-economy/consumption.yaml

```
@@ -1,5 +1,22 @@
 - Consumption:
-    description: total consumption of all goods, by all consumers in a region
+    description: Total consumption of all goods, by all consumers in a region
     unit: billion USD_2010/yr
     tier: 1
     lower_bound: 0
+- Consumption|{Population Deciles} [Share]:
+    description: Share of total consumption accruing to the {Population Deciles}
+    unit: '%'
+    tier: 2
+    higher_bound: 100
+    lower_bound: 0
```

Technically, upper and lower bound are equivalent to range, but better to use the shorter, more intuitive version as default...
```suggestion
    range: [ 0, 100 ]
```